selftests/run_coverage

```diff
@@ -6,7 +6,9 @@
 # Author: Lukas Doktor <ldoktor@redhat.com>
 
 coverage erase
-coverage run --include "avocado/*" ./selftests/run
+rm .coverage.*
```

Comment:
the `.coverage` is erased by `coverage erase` while the `.coverage.*` files are created by the parallel-executed coverage executions.